probe/kubernetes/client.go

```diff
@@ -15,6 +15,7 @@ import (
 	apibatchv2alpha1 "k8s.io/api/batch/v2alpha1"
 	apiv1 "k8s.io/api/core/v1"
 	apiextensionsv1beta1 "k8s.io/api/extensions/v1beta1"
+	storage_v1 "k8s.io/api/storage/v1"
```

Comment:
Will you please replace `storage_v1` with `storagev1` because  I think we should use the same methodology of Weave Scope to declare variables.